core/EE_Session.core.php

```diff
@@ -194,11 +204,11 @@ public static function instance(
         // session loading is turned ON by default, but prior to the init hook, can be turned back OFF via:
         // add_filter( 'FHEE_load_EE_Session', '__return_false' );
         if (! self::$_instance instanceof EE_Session
-            && apply_filters('FHEE_load_EE_Session', true)
             && $cache_storage instanceof CacheStorageInterface
             && $lifespan instanceof SessionLifespan
             && $request instanceof RequestInterface
             && $session_start_handler instanceof SessionStartHandler
+            && apply_filters('FHEE_load_EE_Session', true)
```

Comment:
just moved this down to the bottom of this list, because the filter takes much more time to complete than any of the instance checks, so if any of those fail, we can break out of the conditional sooner.